### DISCUSSION TITLE
Welcome to firebase-js-sdk Discussions!

### DISCUSSION THREAD

**Discussion by samtstern** [firebase/firebase-js-sdk#4360] [General]
## 👋 Welcome!
  We’re using Discussions as a place to get feedback on our new APIs and have high-level conversations that don't belong in Issues.

You can:
  * Share ideas.
  * Engage with other community members.
  * Give us open-ended feedback about our SDK.

There are only two rules:

1. **Welcome others and be open-minded.** Firebase is a community!
1. **Stay on topic.** This repository is only for the Firebase JS SDK and is not the place to discuss general Firebase issues (StackOverflow, Reddit, etc are better forums for that)

**Comment by 816013060** [comment]
Goodnight everyone, im currently having an issue where whenever i try to deploy a function i get the following error 
![image](https://user-images.githubusercontent.com/71352120/158540455-f4490350-0303-453c-adf0-5b5940780358.png). I have tried redeploying a function that successfully deployed before and i am receiving the same error. This weird behavior started after i updated the CLI after i was prompted to.

**Reply by hsubox76** [reply]
You may be more likely to get an answer if you ask in https://github.com/firebase/firebase-tools (Firebase CLI) or https://github.com/firebase/firebase-functions (the Firebase Functions library) as they contain the code for deploying (firebase-functions) and writing (firebase-functions) functions. This repo (firebase-js-sdk) only has the code for calling httpsCallable functions from the client.
